### PR TITLE
Add `play playlist` command

### PR DESCRIPTION
**Pull request by ryansch**
Use case: A user already has a playlist link/uri from browsing their own spotify account. They want to command the bot to play that playlist.

Usage:
`bot play playlist spotify:playlist:0e1zsySwa4K8yu96GIBVZq`
`bot play playlist https://open.spotify.com/playlist/0e1zsySwa4K8yu96GIBVZq?si=IBUU5MRQR6O5suZFVRyNgQ`

Screen Shot 2020-01-18 at 8 26 04 AM

### CODE REVIEW

File: includes/bot_regex.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 class BotRegex < Regexp
   def initialize(*values)
     values = values.map { |value| Regexp.new(value).source }.join('|')
-    super("^(?<bot>[[:alnum:][:punct:]@<>]*)[\\s]+(?<command>#{values})([\\s]+(?<expression>.*)|)$", Regexp::IGNORECASE)
+    super("^#{bot_matcher}[\\s]+(?<command>#{values})([\\s]+(?<expression>.*)|)$", Regexp::IGNORECASE | Regexp::MULTILINE)
```
Comment (ryansch) [inline includes/bot_regex.rb:4]: From https://github.com/slack-ruby/slack-ruby-bot/commit/ee4c57b246acdda17325a1263976a13bca385f3f

**Review by estiens — CHANGES_REQUESTED** [review]
Great thanks!

One suggestion - can we leave the syntax for `play` the same and simple use playlist <url> for the play list command rather than the more verbose `play music` and `play playlist`

**Comment by ryansch** [comment]
@estiens Hasn't the `play music` command always been that way? The help spits out `play music` so it never occurred to me to try `play` by itself.

**Comment by estiens** [comment]
oh, you might be right, it's been so long since I've used this or looked at
the code that I might have the wrong memories! Thanks for the improvements.
I'll pull it down soon and then get merged to master!

**Pull request merged**